docs/application/native/guides/app-management/app-resources.md

```diff
@@ -109,20 +115,49 @@ app_terminate(void *data)
 
 To check how the resources are shown on different device configurations, create emulator instances with different resolutions with the [Emulator Manager](../../../tizen-studio/common_tools/emulator-manager.md) and run the application on the instances using different language settings:
 
-1. Open the Emulator Manager by selecting **Tools &gt; Emulator Manager** in the Tizen Studio menu. ![Emulator Manager](./media/resource_manager_conn_explorer.png)
+1. Open the Emulator Manager by selecting **Tools &gt; Emulator Manager** in the Tizen Studio menu.
+
+  ![Emulator Manager](./media/resource_manager_conn_explorer.png)
+
+2. In the **Emulator Manager** window, the HD Mobile emulator has already been created.Next, create an emulator instance with a WVGA resolution by clicking **Create**.
 
-2. In the **Emulator Manager** window, the HD Mobile emulator has already been created.Next, create an emulator instance with a WVGA resolution by clicking **Create**.![Create a new emulator](./media/resource_manager_emulator_new.png)Select mobile, and the **WVGA Mobile** template. Click **Finish**. If the HD Mobile emulator has not been created, create it as well. For more information on creating an emulator instance, see [Creating Emulator Instances](../../../tizen-studio/common_tools/emulator-manager.md#create).
+  ![Create a new emulator](./media/resource_manager_emulator_new.png)
+
+  Select mobile, and the **WVGA Mobile** template. Click **Finish**. If the HD Mobile emulator has not been created, create it as well. For more information on creating an emulator instance, see [Creating Emulator Instances](../../../tizen-studio/common_tools/emulator-manager.md#create).
```

Comment:
Broken link because of `common_tools`. `../../../tizen-studio/common-tools/emulator-manager.md#create` is correct.

Reply:
Done